ext/rotoscope/rotoscope.c

```diff
@@ -275,6 +282,7 @@ VALUE rotoscope_stop_trace(VALUE self)
   if (rb_tracepoint_enabled_p(config->tracepoint))
   {
     rb_tracepoint_disable(config->tracepoint);
+    config->state = RS_OPEN;
```

Comment:
Document what?  The behaviour of this method hasn't really changed, other than now having to change the state away from the new RS_TRACING state.